server/src/web_rtc_signaling_server.ts

```diff
@@ -102,12 +102,28 @@ export class WebRTCSignalingServer {
   ): Promise<void> {
     await this.connectionStore.addConnection(log, walletAddress, connection);
     log.info(`addUserConnection walletAddress: ${walletAddress}}`);
+    await sendAnalytics(
+      {
+        clientId: walletAddress,
+        category: "Login",
+        action: "Connected",
+      },
+      log,
+    );
   }
 
   public async removeUserConnection(log: FastifyLoggerInstance, walletAddress: string) {
     log.info(`removeUserConnection connection walletAddress: ${walletAddress}`);
     await this.connectionStore.removeFromWaitingList(log, walletAddress);
     await this.connectionStore.deleteConnection(log, walletAddress);
+    await sendAnalytics(
```

Comment:
await Promise.all([]) these three promises so they can run in parallel

Reply:
fixed